app/src/main/java/com/fstyle/androidtrainning/data/remote/service/config/LastFmApi.java

```diff
@@ -0,0 +1,48 @@
+package com.fstyle.androidtrainning.data.remote.service.config;
+
+import com.fstyle.androidtrainning.data.remote.service.response.AlbumSearchResponse;
+import com.fstyle.androidtrainning.data.remote.service.response.ArtistSearchResponse;
+import com.fstyle.androidtrainning.data.remote.service.response.RecentTrackResponse;
+import com.fstyle.androidtrainning.data.remote.service.response.TopAlbumResponse;
+import com.fstyle.androidtrainning.data.remote.service.response.TopTrackResponse;
+import com.fstyle.androidtrainning.data.remote.service.response.TrackSearchResponse;
+import retrofit2.Call;
+import retrofit2.http.GET;
+import retrofit2.http.Query;
+
+/**
+ * Created by Administrator on 12/08/17.
+ */
+
+public interface LastFmApi {
+
+    @GET("/2.0")
+    Call<TopAlbumResponse> getTopAlbum(@Query("method") String method,
+            @Query("user") String nameUser, @Query("api_key") String apiKey,
+            @Query("format") String format);
```

Comment:
WARNING: [AvoidDuplicateLiterals] The String literal "format" appears 6 times in this file; the first occurrence is on line 22
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/strings.html#AvoidDuplicateLiterals